runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightPlugin.java

```diff
@@ -30,7 +30,7 @@
 import net.runelite.client.ui.overlay.Overlay;
 
 @PluginDescriptor(
-	name = "Mouse highlight"
+	name = "Mouse Highlight"
```

Comment:
I think a better name would be "Mouse Tooltips"

Reply:
Agreed, done